university-of-gothenburg-apa-swedish-legislations.csl

```diff
@@ -0,0 +1,697 @@
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <choose>
+          <if type="report">
+            <text variable="publisher"/>
+            <text macro="title"/>
+          </if>
+          <else>
+            <text macro="title"/>
+          </else>
+        </choose>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <choose>
+      <if type="legislation" match="any">
+        <text variable="container-title" suffix=" "/>
+        <text variable="volume"/>
+      </if>
+    </choose>
+    <names variable="author">
+      <name form="short" and="symbol" delimiter=", " initialize-with=". "/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <choose>
+          <if type="report">
+            <text variable="publisher"/>
+            <text variable="title" form="short" font-style="italic"/>
+          </if>
+          <else-if type="legal_case">
+            <text variable="title" font-style="italic"/>
+          </else-if>
+          <else-if type="book graphic  motion_picture song" match="any">
+            <text variable="title" form="short" font-style="italic"/>
+          </else-if>
+          <else-if type="legislation"/>
+          <else-if type="bill" match="any">
+            <text variable="title" form="short" font-style="italic"/>
+          </else-if>
+          <else-if variable="reviewed-author">
+            <choose>
+              <if variable="reviewed-title" match="none">
+                <text variable="title" form="short" font-style="italic" prefix="Review of "/>
+              </if>
+              <else>
+                <text variable="title" form="short" quotes="true"/>
+              </else>
+            </choose>
+          </else-if>
+          <else>
+            <text variable="title" form="short" quotes="true"/>
+          </else>
+        </choose>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if type="thesis report" match="any">
+        <choose>
+          <if variable="DOI" match="any">
+            <text variable="DOI" prefix="https://doi.org/"/>
+          </if>
+          <else-if variable="archive" match="any">
+            <group>
+              <text term="retrieved" text-case="capitalize-first" suffix=" "/>
+              <text term="from" suffix=" "/>
+              <text variable="archive" suffix="."/>
+              <text variable="archive_location" prefix=" (" suffix=")"/>
+            </group>
+          </else-if>
+          <else>
+            <group>
+              <text term="retrieved" text-case="capitalize-first" suffix=" "/>
+              <text term="from" suffix=" "/>
+              <text variable="URL"/>
+            </group>
+          </else>
+        </choose>
+      </if>
+      <else>
+        <choose>
+          <if variable="DOI">
+            <text variable="DOI" prefix="doi: "/>
```

Comment:
this seems odd -- you probably also want the https://doi.org/ prefix here, no?

Reply:
According to our guidelines "doi:" in lowercase is the way to go here.

Please refer to page 6 of our guidelines (although in Swedish) for reference: http://www.utbildning.gu.se/digitalAssets/1366/1366320_apa-lathunden-2012.pdf

Reply:
OK, but then this is incorrect further up: https://github.com/citation-style-language/styles/pull/2779/files#diff-1717d9d6b09f6c2686dcf86ba0e2c385R132 

If you have any way to affect the citation style, btw. using `doi: ` is a bad idea. It stems from a time when people expected it to become its own internet protocol like https: or ftp: The URL form, `https://doi.org/` is now universally preferred.

Reply:
(see also https://www.zotero.org/support/kb/doi_in_apa)